install-linux.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#create tile38 user
+useradd -m -d /var/lib/tile38 -s /sbin/nologin -U tile38
```

Comment:
This should be optional. I do not want a separate tile38 user on my dev machine.

Reply:
yeah you are right, but it's necessary for more security.